web/index.html

```diff
@@ -645,9 +777,20 @@ <h3>Used in</h3>
                         console.error('Error loading metadata:', error);
                     });
 
+                fetch('bookmarklet.js')
+                    .then(response => { return response.text() })
+                    .then(bookmarkletText => {
+                        this.bookmarkletData = bookmarkletText.replace('HELPER_URL', HELPER_PATH);
+                    })
+                    .catch(error => {
+                        console.error('Error loading bookmarklet:', error);
+                    });
```

Comment:
Fetch the code for the bookmarklet and then replace the dummy path with the path we are using based on if this is being run in github or a local server

Reply:
That entire script could just live right here copy pasted, but this is kinda nice, since it would just be a big ugly string block of code

Reply:
Switch to ugly block of code as in earlier comment

Reply:
done